src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -246,8 +246,21 @@ protected void describe(IceUdpTransportPacketExtension pe)
         fingerprintPE.setFingerprint(dtlsStack.getLocalFingerprint());
         fingerprintPE.setHash(dtlsStack.getLocalFingerprintHashFunction());
 
-        // TODO: don't we only support ACTIVE right now?
-        fingerprintPE.setSetup("ACTPASS");
+        final DtlsRole role = dtlsStack.getRole();
+        final String setupRole;
+        if (role instanceof DtlsServer)
```

Comment:
i worry the way this is written is perhaps a bit misleading.  if i'm understanding it correctly, this code will return `actpass` if the role is null (meaning we haven't decided yet) or active/passive after it's already been set as client or server, right?  this makes it look like `role` could be an instance of some other class and that results in it being actpass.  i'm just thinking we could be a bit more explicit and do:

```
final DtlsRole role = dtlsStack.getRole();
final String setupRole;
if (role == null)
{
    // We've not chosen a role yet, so we can be either
    setupRole = "actpass";
}
else if (role instanceof DtlsClient)
{
    setupRole = "active";
}
else if (role instanceof DtlsServer)
{
    setupRole = "passive";
}
else
{
    // throw exception?
}
fingerprintPE.setSetup(setupRole);
```

though part of me also wonders if this may be better done as something like `dtlsStack.describeRole()` as well